.github/workflows/publish-release.yml

```diff
@@ -82,8 +82,8 @@ jobs:
     runs-on: ubuntu-latest
     env:
       CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_TOKEN }}
-      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      GH_TOKEN: ${{ secrets.GH_TOKEN_PAT }}
```

Comment:
Why did this switch to PAT? What did github_token not provide?

Reply:
Using the regular token won't trigger the next workflow (the `release` triggers, in our case) because GitHub doesn't want workflow recursion. They do allow you to use a PAT for this however. 

From their [docs](https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow):
```
If you do want to trigger a workflow from within a workflow run, you can use a GitHub App installation access token or a personal access token instead of GITHUB_TOKEN to trigger events that require a token.
```

Reply:
We followed the advice from [this](https://github.com/orgs/community/discussions/25281) discussion thread that is very similar to our use case.